extensions/blocks/tiled-gallery/layout/index.js

```diff
@@ -3,6 +3,7 @@
  */
 import { __, sprintf } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
+import classnames from 'classnames';
```

Comment:
Confirmed that this doesn't end up to built bundle meant for frontend `_inc/blocks/tiled-gallery/view.js` to make it needlessly bigger.